scripts/g.extension/g.extension.py

```diff
@@ -2299,46 +2332,48 @@ def resolve_source_code(url=None, name=None, branch=None, fork=False):
     >>> resolve_source_code('https://bitbucket.org/joe-user/grass-module') # doctest: +SKIP
     ('remote_zip', 'https://bitbucket.org/joe-user/grass-module/get/default.zip')
     """
+    # Handle URL for the offical repo
     if not url and name:
         module_class = get_module_class_name(name)
-        # note: 'trunk' is required to make URL usable for 'svn export' call
+        repo_branches = get_github_branches(
+            "https://api.github.com/repos/OSGeo/grass-addons/branches",
+            version_only=True,
+        )
+        # Define branch to fetch from (latest or current version)
+        version_branch = "grass{}".format(version[0])
+        if version_branch not in repo_branches:
+            version_branch = repo_branches[-1]
```

Comment:
Just add a comment here that we assume the official repo always has at least one version branch. May be useful if this code gets reused for something else.